src/components/StackedAvatars/styles.tsx

```diff
@@ -0,0 +1,28 @@
+import styled from "styled-components"
+
+const AVATAR_SIZE = 32
+const OVERLAP_AMOUT = 0.4 // 0 to 1
+
+function containerWidth(count: number): number {
+  if (count === 0) {
+    return 0
+  }
+
+  return AVATAR_SIZE + count * AVATAR_SIZE * (1 - OVERLAP_AMOUT)
+}
+
+export const StackedAvatarsWrapper = styled.div<{ count: number }>`
+  display: flex;
+  margin-right: 16px;
+  width: ${props => containerWidth(props.count)}px;
+`
+
+export const AuthorAvatar = styled.img<{ index: number }>`
+  position: relative;
+  width: ${AVATAR_SIZE}px;
+  height: ${AVATAR_SIZE}px;
+  border: 2px solid #fff;
```

Comment:
I think we should be grabbing the border color from a background-color variable here so it doesn't cause problems when implement dark mode.

Reply:
Can probably do that when we set up dark mode. There's a bunch of things in consideration, so probably out of scope of this.